src/pushsource/_impl/model/ami.py

```diff
@@ -1,46 +1,20 @@
-import datetime
-
 from frozenlist2 import frozenlist
 
-from .base import PushItem
 from .. import compat_attr as attr
-from .conv import datestr, instance_of_str, instance_of, optional_str, optional
-
-
-@attr.s()
-class AmiRelease(object):
-    """Release metadata associated with an AMI."""
-
-    product = attr.ib(type=str, validator=instance_of_str)
-    """A short product name, for example "RHEL" if this is an image for Red Hat
-    Enterprise Linux."""
-
-    date = attr.ib(type=datetime.date, converter=datestr)
-    """Date at which this image was generated."""
-
-    arch = attr.ib(type=str, validator=instance_of_str)
-    """Architecture of the image, for example "x86_64"."""
-
-    respin = attr.ib(type=int, validator=instance_of(int))
-    """Respin count. 0 for original build of an image, incremented by one for
-    each rebuild."""
+from .conv import instance_of_str, instance_of, optional_str, optional
+from .vms import VMIRelease, VMIPushItem
 
-    version = attr.ib(type=str, default=None, validator=optional_str)
-    """A <major>.<minor> version string for the image's product version, for example
-    "7.9" if this is an image for Red Hat Enterprise Linux 7.9.x."""
 
-    base_product = attr.ib(type=str, default=None, validator=optional_str)
-    """For layered products, name of the base product for which the image should be used."""
-
-    base_version = attr.ib(type=str, default=None, validator=optional_str)
-    """For layered products, version of the base product for which the image should be used."""
+class AmiRelease(VMIRelease):
+    """
+    Release metadata associated with an AMI.
 
-    variant = attr.ib(type=str, default=None, validator=optional_str)
-    """Variant of this image's product (only for products which have variants).  For example,
-    "Server", for Red Hat Enterprise Linux Server."""
+    This class has the same attributes ass VMIRelease and its only kept
+    for external tools compatibility.
```

Comment:
Just say "backwards compatibility", it is enough justification to not remove something from an API.


```suggestion
    for backwards compatibility.
```

Reply:
Fixed